test/test_dataDict.py

```diff
@@ -26,6 +26,14 @@ def test_dataDict():
     amplm = {}
     omegalm = {}
     dt = dataDict["t"][1] - dataDict["t"][0]
+    # check that the time array is uniform. This is required to obtain omega by
+    # taking derivative of phase applying time_deriv_4thorder.
+    ids = np.where(np.abs(np.diff(dataDict["t"]) - dt) > 1e-7)[0]
+    if len(ids) > 0:
+        dt_ = np.diff(dataDict["t"])[ids[0]]
+        raise Exception("t_zeroecc is not uniform. dt at the start of the "
+                        f"time array is {dt}. dt={dt_} between {ids[0]} and "
+                        f"{ids[0]+1}. The difference is {dt - dt_}")
```

Comment:
You already did a better version of this check here: https://github.com/vijayvarma392/gw_eccentricity/blob/main/gw_eccentricity/eccDefinition.py#L192, just repeat it here? Or make it a function if you like.

Reply:
Done